### ISSUE TITLE
train_test_split shuffle problem

### ISSUE THREAD

**Issue by soloblack** [pycaret/pycaret#68]
First of all, my English is not very good, may not express clearly. When the setup() function is executed, it is used to split the training set and the test set, shuffle param is default True. This is fine in most scenarios, but it can cause serious overfitting problems in timeseries tasks.

**Comment by pycaret** [comment]
Will include shuffle parameter in setup() in future release.

Marking this as enhancement.

**Comment by pycaret** [comment]
Feature now available in pycaret-nightly.
You can install pycaret-nightly using "pip install pycaret-nightly".

New parameter folds_shuffle is now added in setup. By default it is set to False.

Can you please test this and close the issue if the enhancement addresses the original issue.

It will be shipped to PyCaret 2.0x in few weeks.

**Comment by soloblack** [comment]
Thx
